core/src/main/java/smile/classification/RBFNetwork.java

```diff
@@ -156,7 +156,7 @@ public Trainer(Metric<T> distance) {
          * @param rbf the radial basis function.
          * @param m the number of basis functions.
          */
-        public Trainer setRBF(RadialBasisFunction rbf, int m) {
+        public Trainer<T> setRBF(RadialBasisFunction rbf, int m) {
```

Comment:
What's the warning about this? My IDE never report warnings about it. And previously another community member scanned the whole code base with lint tools. He didn't report something like this. I am just wonder why and is it because of Java 8? Thanks!